public void onFailure(@AttentionFailureCodes int error){
    try {
        mCallback.onFailure(error);
    } catch (RemoteException e) {
        e.rethrowFromSystemServer();
    }
}